Jkong27/users page

jkong27 opened the pull request:
pull request for users page

EvanELC committed:
users from api

EvanELC committed:
updated base

EvanELC committed:
forgot parentheses on function call

josephkong13 committed:
API call working?

EvanELC committed:
all users added to basic list on webpage

josephkong13 committed:
added logic on whether to display username or full name

josephkong13 committed:
displays each user in the team, but needs correct styling

josephkong13 committed:
fixed styling

wtpisaac commented:
Looks good, I'm going to make some refactors to the internal API to refer to Member objects as "Members" and Owners as "Users" to keep it more in line with the backend. (I know the UX wording is Users; might change that but internally we should probably use the names defined on the backend)

wtpisaac committed:
Rename some items from Owner to User

Ideally, the User of a Member will refer to the Django User object.
This should be made consistent on the backend. Currently, there is
an 'owner' of a member, which feels a bit ambiguous/misleading.
However, until this is changed on the backend, it will remain this
way on the frontend.

wtpisaac committed:
Removed image from Users screen (for now)
